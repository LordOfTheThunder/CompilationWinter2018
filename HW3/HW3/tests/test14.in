void TestSameNameFunc() {
	int x;
}
void TestSameNameFunc() {
	int x;
}
