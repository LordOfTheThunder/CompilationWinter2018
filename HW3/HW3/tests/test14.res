---end scope---
x INT 0
line 4: identifier TestSameNameFunc is already defined
