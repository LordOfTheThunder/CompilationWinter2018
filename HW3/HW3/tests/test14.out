---end scope---
x INT 1
line 4: identifier TestSameNameFunc is already defined
